python/vyos/template.py

```diff
@@ -172,6 +172,14 @@ def force_to_list(value):
     else:
         return [value]
 
+@register_filter('prepend_str')
+def prepend_str(value, prefix):
```

Comment:
Why kot use the Jinja2 ~ operator to concat strings?

Reply:
We could.

I thought this was cleaner and would be easier to maintain than @sarthurdev’s suggestion here:
> Need to change to: `cacerts = {{ '.pem,'.join(rw_conf.authentication.x509.ca_certificate) ~ '.pem' }}`

(in particular, the use of `join` with `.pem` as the separator is clever, but IMHO not immediately obvious what it’s doing)

Reply:
Please proceed with join and pem separator and add a proper comment.

Reply:
Updated and confirmed smoke tests still pass.